forecast-bbs-core.R

```diff
@@ -28,18 +28,36 @@ install_dataset <- function(dataset){
 #'
 #' Removes waterbirds, shorebirds, owls, kingfishers, knightjars,
 #' dippers. These species are poorly sampled due to their aquatic or
-#' noctural nature.
+#' noctural nature. Also removes taxa that were either partially unidentified 
+#' (e.g. "sp.") or were considered hybrids (e.g. "A x B").
 #'
 #' @param df dataframe containing an species_id column
 #'
 #' @return dataframe, filtered version of initial dataframe
 filter_species <- function(df){
-  bbs_data_filtered <- df %>%
+  
+  con <- dbConnect(RPostgres::Postgres(), dbname = 'postgres', 
+                   host = "localhost", password = "p", user = "postgres")
```

Comment:
Oops, yeah.  Good catch. I finally got RPostgres to recognize my host (with PGHOST) and password (with . pgpass), but it still defaults to using "davidharris" as my username even though:

* I have `export PGUSER=postgres` in my .bash_profile
* When I run `psql` from the command line, it respects PGUSER

Anyway, I'll push a new commit without the credentials.